From f8ed78f1b2e65f2741455612d38477424e321850 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Thu, 15 Feb 2024 20:49:27 +0800
Subject: [PATCH] LoongArch: Fix a build error with GCC 14 (#825)

Fix the build error with GCC 14:

    ../src/loongarch64/ffi.c: In function 'ffi_prep_closure_loc':
    ../src/loongarch64/ffi.c:525:7: error: implicit declaration of
    function 'ffi_tramp_set_parms' [-Wimplicit-function-declaration]
---
 src/loongarch64/ffi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/loongarch64/ffi.c b/src/loongarch64/ffi.c
index 140be3bc3..b5ac18ea9 100644
--- a/src/loongarch64/ffi.c
+++ b/src/loongarch64/ffi.c
@@ -28,6 +28,7 @@
 
 #include <ffi.h>
 #include <ffi_common.h>
+#include <tramp.h>
 
 #include <stdlib.h>
 #include <stdint.h>
